index.html

```diff
@@ -0,0 +1,161 @@
+
+        </section>
+
+        <section id="mainProgram">
+            <h3 id="mainProgramTitle" class="subTitle">Main Program</h3>
+            <img class="underline" src="img/text/underline.svg" alt="underline">
+            <div id="cardsContainer">
+                <ul id="cardsListUL">
+                    <li class="mainProgramCards">
+                        <img class="cardIcon" src="img/icons/lecture.svg" alt="Lecture icon">
+                        <h4 class="mainProgramCardsTitle">Lecture</h4>
+                        <p class="mainProgramCardsP">Listen to the speakers from various countries about the messages of
+                            sharing and opening.</p>
+                    </li>
+                    <li class="mainProgramCards">
+                        <img class="cardIcon" src="img/icons/CC exibition.svg" alt="CC Exibition icon">
+                        <h4 class="mainProgramCardsTitle">CC Exibition</h4>
+                        <p class="mainProgramCardsP">Appreciate various creations applying CC license of artists,
+                            organized from Art Center Nabi</p>
+                    </li>
+                    <li class="mainProgramCards">
+                        <img class="cardIcon" src="img/icons/Forum.svg" alt="Forum icon">
+                        <h4 class="mainProgramCardsTitle">Forum</h4>
+                        <p class="mainProgramCardsP">Have the time to share your thoughts and opinions with experts for
+                            each topic.</p>
+                    </li>
+                    <li class="mainProgramCards">
+                        <img class="cardIcon" src="img/icons/Workshop.svg" alt="Workshop icon">
+                        <h4 class="mainProgramCardsTitle">Workshop</h4>
+                        <p class="mainProgramCardsP">Try creating your own work using open source license rather than
+                            just watching at it.</p>
+                    </li>
+                    <li class="mainProgramCards">
+                        <img class="cardIcon" src="img/icons/CC ignite.svg" alt="CC Ignite icon">
+                        <h4 class="mainProgramCardsTitle">CC Ignite</h4>
+                        <p class="mainProgramCardsP">get opportunities to network with CC affiliates around the world,
+                            also after the summit.</p>
+                    </li>
+                </ul>
+            </div>
+            <button id="joinButton"> Join CC Global Summit 2015</button>
+            <div id="endTitle">SEE THE WHOLE PROGRAM</div>
+
+
+        </section>
+
+        <section id="featured">
+            <h3 class="featuredSubTitle">Featured Speakers</h3>
+            <img class="underline" src="img/text/underline.svg" alt="underline">
+
+            <div  class="gridContainer">
+           
+            </div>
+
+            <button id="more">MORE <img id="orangeArrow" src="img/text/orangeArrow.svg" alt="Orange Arrow"></button>
```

Comment:
- [ ] As it was raised earlier, to avoid having redundant element in the UI; kindly remove this `button `since it's no longer listeninng to `click` event.. 👍 **Redundant elements can clutter the UI, confuse users, and negatively impact the design's effectiveness.**

<img width="262" alt="Untitled" src="https://github.com/DamianDib9/MV-Capstone-project/assets/103386183/75825505-61d6-4563-b853-7ab202a00064">

Reply:
Hello @DamianDib9 👍